features-json/picture-in-picture.json

```diff
@@ -337,8 +337,7 @@
   },
   "notes":"",
   "notes_by_num":{
-    "1":"Apple provides an equivalent proprietary API for Picture-in-Picture.",
-	"2":"Enabled in Chrome through the \"#enable-picture-in-picture\" and \"#enable-surfaces-for-videos\" flag in chrome://flags",
+    "1":"Apple provides an equivalent proprietary API for Picture-in-Picture."
```

Comment:
You missed a comma

Reply:
@Maxou44 (gentle ping)

Reply:
```suggestion
    "1":"Apple provides an equivalent proprietary API for Picture-in-Picture.",
```

Reply:
Thanks for the fix 👌

Reply:
@Maxou44 I think you need to actually accept/make the suggested fix, I'm not seeing it as a PR commit.